Sources/MapboxNavigation/CarPlayMapViewController.swift

```diff
@@ -190,16 +207,52 @@ public class CarPlayMapViewController: UIViewController {
         styleManager?.styles = styles
     }
     
+    func setupSpeedLimitView() {
+        let speedLimitView = SpeedLimitView()
+        speedLimitView.translatesAutoresizingMaskIntoConstraints = false
+        view.addSubview(speedLimitView)
+        
+        speedLimitView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor, constant: 8).isActive = true
+        speedLimitView.rightAnchor.constraint(equalTo: view.rightAnchor, constant: -8).isActive = true
+        speedLimitView.widthAnchor.constraint(equalToConstant: 30).isActive = true
+        speedLimitView.heightAnchor.constraint(equalToConstant: 30).isActive = true
```

Comment:
nit: There is a nice convenience method for activating multiple constraints:
```suggestion
        NSLayoutConstraint.active([
            speedLimitView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor, constant: 8),
            speedLimitView.rightAnchor.constraint(equalTo: view.rightAnchor, constant: -8),
            speedLimitView.widthAnchor.constraint(equalToConstant: 30),
            speedLimitView.heightAnchor.constraint(equalToConstant: 30),
        ])
```

Reply:
I prefer the `isActive = true` approach because it takes fewer lines 😀

Reply:
Both approaches work for me, I don't see major advantages/disadvantages between the two though.